### DISCUSSION TITLE
where is the place of using root layout file in electron template?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [vercel/next.js#52118] [Help] [answered]
### Summary

I use below command created the nextjs with electron template project:
```
npx create-next-app --example with-electron-typescript electron-nextjs-demo-01
```

now the project directories is below:
![image](https://github.com/vercel/next.js/assets/12727582/3a759a34-0a45-446c-a2ca-208bb48ddeda)

the default root Layout file is: `renderer/components/Layout.tsx`

but I have several questions about it:

as we know, next.js use page  routing. 

---

Contrast to the next.js web, I can use `pages/_app.tsx` to wrapper the Layout, 
![image](https://github.com/vercel/next.js/assets/12727582/b9f1589e-8a9d-4475-aeb2-81cfe02acfc0)

but in this case, how to wrapper Layout in the next.js with electron projecy?

**Comment by icyJoseph** [comment, marked as answer]
Have you tried adding a `_app.tsx` under `examples/with-electron-typescript/renderer/pages`? what happens if you do?
